app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt

```diff
@@ -1333,8 +1333,9 @@ class HomeActivityTest {
     }
   }
 
+  @Config(qualifiers = "+port")
   @Test
-  fun testHomeActivity_allTopicsCompleted_displaysAllTopicCards() {
+  fun testHomeActivity_allTopicsCompleted_mobilePortraitDisplaysAllTopicCardsIn2Columns() {
```

Comment:
```suggestion
  fun testHomeActivity_allTopicsCompleted_mobilePortrait_displaysAllTopicCardsIn2Columns() {
```

Reply:
In this test, since the test name specifies mobile landscape, there is no need for checking `if (context.resources.getBoolean(R.bool.isTablet))` as that is now a different test.

The same applies to the rest of the tests.